docs/api.yml

```diff
@@ -0,0 +1,185 @@
+                  status:
+                    type: string
+  /queue/{queue_name}/delete:
+    post:
+      summary: Delete the queue with the specified queue_name
+      parameters:
+        - name: queue_name
+          in: path
+          schema:
+            type: string
+          required: true
+      responses:
+        200:
+          description: Request processed
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  status:
+                    type: string
+  /queues.json:
+    get:
+      summary: List Queues and their state
+      responses:
+        200:
+          description: |
+            Return object containting list of queues and theis states
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  queues:
+                    type: array
+                    items:
+                      type: object
+                      properties:
+                        name:
+                          type: string
+                        count:
+                          type: integer
+                        url:
+                          type: string
+                          format: url
+  /jobs/{queue_name}/{page}.json:
+    get:
+      summary: List workers and their state
+      parameters:
+        - name: queue_name
+          in: path
+          schema:
+            type: string
+          required: true
+        - name: page
+          in: path
+          schema:
+            type: integer
+          required: true
+      responses:
+        200:
+          description: |
+            Return object containting list of worker states in workers property 
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  name:
+                    type: string
+                  jobs:
+                    type: array
+                    items:
+                      type: object
+                      properties:
+                        id:
+                          type: string
+                          format: uuid
+                        status:
+                          type: string
+                        enqueued_at:
+                          type: string
+                          format: date-time
+                        started_at:
+                          type: string
+                          format: date-time
+                        origin:
+                          type: string
+                        error_message:
+                          type: string
+                        description:
+                          type: string
+                  pagination:
+                    type: object
+                    properties:
+                      pages_in_window:
+                        type: integer
```

Comment:
What does this number represent?

Reply:
current production api doesn't return number but 
```json
{
  "pagination": {
    "pages_in_window": [
      { "number": 1, "url": "/admin/rt_dashboard/inner/%5Bfailed%5D/1" }
    ]
  }
}
```

Reply:
ok i'll have to check